app/build.gradle

```diff
@@ -491,4 +494,43 @@ afterEvaluate {
             }
         }
     }
+
+    // Format test output. Ported from AC #2401
+    tasks.matching {it instanceof Test}.all {
+        testLogging.events = []
```

Comment:
You might want to consider adding those system properties to also get the logs from Robolectric and AC printed:
https://github.com/mozilla-mobile/android-components/blob/master/build.gradle#L71-L72